examples/channel-directions/channel-directions.go

```diff
@@ -1,21 +1,19 @@
-// When using channels as function parameters, you can
-// specify if a channel is meant to only send or receive
-// values. This specificity increases the type-safety of
-// the program.
+// Quando passi un channel come parametro di funzione, puoi indicare
+// se il channel deve essere utilizzato solamente per ricevere o per inviare messaggi.
+// Questa indicazione aumenta la _type-safety_ dell'intero programma.
 
 package main
 
 import "fmt"
 
-// This `ping` function only accepts a channel for sending
-// values. It would be a compile-time error to try to
-// receive on this channel.
+// Questa funzione `ping` accetta un channel per il solo invio.
+// Provare a ricevere un valore su questo canale genererà un errore a compile-time
```

Comment:
Io in genere non si usa `nel compile-time` [wikipedia](https://it.wikipedia.org/wiki/Compile-time). Secondo me possiamo scegliere fra:

- A compile time
- A tempo di compilazione

che famo?